### ISSUE TITLE
Simplify sector confirmation message flow 

### ISSUE THREAD

**Issue by ZenGround0** [filecoin-project/specs-actors#475]
Currently `Power.OnSectorProveCommit` is hit by way of a Power -> Miner -> Power cycle.  From builtin/miner/miner_actor.go:ConfirmSectorProofsValid:

```
		// TODO: since this method gets called by the storage power actor
		// initially, can we just do this while we're there?
		// We can probably return the right information from this call to the caller, so it can update there
		// TODO: we should batch these calls...
```

We should evaluate whether it is better to return info needed for power.OnSectorProveCommit back up to the power cron tick per the first TODO

Per the second TODO we should eval doing batch send either in the current configuration or the new one if we end up addressing the first TODO.

**Comment by anorth** [comment]
The flow here has been simplified a lot since the TODO was written. I don't think we should go so far as trying to return info to the power actor to save one call, but we should do all the batching we can.

**Comment by anorth** [comment]
I think you probably need to do this alongside #476

**Comment by ZenGround0** [comment]
Closed by #577
